http/help_test.go

```diff
@@ -14,8 +15,9 @@ func TestHelp(t *testing.T) {
 	TestServerAuth(t, addr, token)
 
 	resp := testHttpGet(t, "", addr+"/v1/sys/mounts?help=1")
-	if resp.StatusCode != http.StatusBadRequest {
-		t.Fatal("expected bad request with no token")
+	fmt.Println(resp.StatusCode)
```

Comment:
Is this needed?

Reply:
Will remove the print statement. Was using it to debug